app/timetables/models.py

```diff
@@ -1,10 +1,12 @@
 from __future__ import unicode_literals
 
 from django.db import models
+
 from app.shared.models import CommonInfo
 
-class MealOption(CommonInfo, models.Model):
-    name = models.CharField(max_length=120)
+
+class MealOption(CommonInfo):
```

Comment:
Let's test this 

You can follow this guide on setting up your testing folder
https://realpython.com/blog/python/testing-in-django-part-1-best-practices-and-examples/